microsoft-edge/webdriver-chromium/ie-mode.md

```diff
@@ -0,0 +1,222 @@
+
+
+<!-- ====================================================================== -->
+## Automate IE mode in Microsoft Edge
+
+The following sections walk you through using Selenium to automate IE mode in Microsoft Edge.
+
+> [!NOTE]
+> This article provides instructions for using the Selenium framework, but you can use any library, framework, and programming language that supports WebDriver.  To accomplish the same tasks using another framework, consult the documentation for your framework of choice.
+
+To launch Microsoft Edge in IE mode with IEDriver:
+
+1.  Create an `InternetExplorerDriverService`.
+
+1.  Define `InternetExplorerOptions` with additional capabilities that point to the Microsoft Edge browser.
+
+1.  Start an instance of `InternetExplorerDriver` and pass it `InternetExplorerDriverService` and `InternetExplorerOptions`.  IEDriver launches Microsoft Edge and then loads your web content in IE mode.
+
+The next section shows the complete sample, and then the subsequent sections focus on each of the main steps that are listed above.
+
+
+<!-- ====================================================================== -->
+## The complete sample
+
+The following sample launches Microsoft Edge in IE mode, navigates to [bing.com](https://www.bing.com/) and searches for "WebDriver".
+
+### [C#](#tab/c-sharp/)
+
+<a id="drive-ie-mode-entire-sample"></a>
+
+```csharp
+using System;
+using OpenQA.Selenium;
+using OpenQA.Selenium.IE;
+
+namespace Selenium_Web
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //change the path accordingly
+            var dir = @"C:\<path to IEDriver download>";
+            var driverexe = "IEDriverServer.exe";
+
+            var ieService = 
+                 InternetExplorerDriverService.CreateDefaultService(dir, driverexe);
+            var ieOptions = new InternetExplorerOptions { IgnoreZoomLevel = true };
+            ieOptions.AddAdditionalCapability("ie.edgechromium", true);
+            //change the path accordingly
+            ieOptions.AddAdditionalCapability("ie.edgepath", 
+                    "C:/Program Files (x86)/Microsoft/Edge/Application/msedge.exe");
+
+            InternetExplorerDriver reqDriver = 
+                                   new InternetExplorerDriver(ieService, ieOptions);
+            reqDriver.Url = "https://bing.com";
+            reqDriver.FindElementById("sb_form_q").SendKeys("WebDriver");
+            reqDriver.FindElementById("sb_form").Submit();
+
+            Console.WriteLine("Done");
+        }
+    }
+}
+```
+
+### [Python](#tab/python/)
+
+<a id="drive-ie-mode-entire-sample"></a>
+
+<!-- todo -->
+Insert Python sample here
+
+* * *
+
+The following sections explain the steps in this sample in more detail.
+
+
+<!-- ====================================================================== -->
+## Create an InternetExplorerDriverService
```

Comment:
Creating the service yourself isn't necessary for most users, and doing so complicates the sample. If you create your own service like this, you need to shut it down manually as well so that the driver executable doesn't stick around after your test. I'd prefer to remove this step and direct users to put IEDriver.exe on the path instead.